app/services/katello/organization_creator.rb

```diff
@@ -26,30 +26,37 @@ def initialize(organization)
     end
 
     def seed!
-      @organization.setup_label_from_name
-      @organization.save!
-
-      create_library_environment
-      create_library_view
-      create_library_cvv
-      create_content_view_environment
-      create_anonymous_provider
-      create_redhat_provider
+      ActiveRecord::Base.transaction do
```

Comment:
Won't this fail if `OrganizationCreator` is run from `102-organizations.rb` within the transaction in `foreman_seeder`?  That's why my first thought was to put the `.transaction` in the controller rather than here, to ensure that we're not trying to nest the transactions.  I'm sure there's a better way I'm just not thinking of, though..

Reply:
If you mean that simply using a transaction here will raise an error - no error!